pkg/plugins/queue/azqueue/azqueue_test.go

```diff
@@ -0,0 +1,373 @@
+// Copyright 2021 Nitric Pty Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package azqueue_service
+
+import (
+	"fmt"
+	//"bytes"
+	//"fmt"
+	"time"
+
+	azqueue2 "github.com/Azure/azure-storage-queue-go/azqueue"
+	"github.com/nitric-dev/membrane/pkg/plugins/queue"
+
+	//"io/ioutil"
+	//"strings"
```

Comment:
Minor tidy up to remove unwanted comments.

Reply:
done